docs/agent/changes.md

```diff
@@ -98,27 +95,80 @@ The agent has a new set of command-line options:
+```
+
+### Linux
+
+There are a few major changes:
+
+* only the _lifecycle commands_ (i.e. `start`/`stop`/`restart`/`status` on the Agent service) should be run with `sudo service`/`sudo initctl`/`sudo systemctl`
+* all the other commands need to be run with the `datadog-agent` command, located in the `PATH` (`/usr/bin`) by default. The binary `dd-agent` is not available anymore.
+* the `info` command has been renamed `status`
+* the Agent 6 does not ship a SysV-init script (previously located at `/etc/init.d/datadog-agent`)
+* The configuration GUI is disabled by default on Linux-based systems.
+
+Most of the commands didn't change, for example this is the list of the _lifecycle commands_
+on Ubuntu:
+
+| Command  | Notes |
+| -------- | ----- |
+| `sudo service datadog-agent start` | Start the Agent as a service |
+| `sudo service datadog-agent stop` | Stop the Agent service |
+| `sudo service datadog-agent restart` | Restart the Agent service |
+| `sudo service datadog-agent status` | Print the status of the Agent service |
+
+Some functionalities are now provided by the Agent binary itself as sub-commands and there's
+no need anymore to invoke them through `service` (or `systemctl`). For example, for an Agent installed on Ubuntu, the differences are as follows:
+
+| Agent5 Command | Agent6 Command | Notes |
+| -------------- | -------------- | ----- |
+| `sudo service datadog-agent info` | `sudo datadog-agent status` | Status page of a running Agent |
+| `sudo service datadog-agent flare` | `sudo datadog-agent flare` | Send flare |
+| `sudo service datadog-agent` | `sudo datadog-agent --help` | Display Agent usage |
+| `sudo -u dd-agent -- dd-agent check <check_name>` | `sudo -u dd-agent -- datadog-agent check <check_name>` | Run a check |
+
+**NB**: If `service` is not available on your system, use:
+
+* on `upstart`-based systems: `sudo start/stop/restart datadog-agent`
+* on `systemd`-based systems: `sudo systemctl start/stop/restart datadog-agent`
+
+### Windows
+
+There are a few major changes
+* the main executable name is now `agent.exe` (it was `ddagent.exe` previously)
+* Commands should be run with the command line `c:\program files\datadog\datadog-agent\embedded\agent.exe <command>`
+* The configuration GUI is now a web-based configuration application, point a
+web browser to `http://localhost:5002` to access it.
 
 ### MacOS
 
 * the _lifecycle commands_ (former `datadog-agent start`/`stop`/`restart`/`status` on the Agent 5) are replaced by `launchctl` commands on the `com.datadoghq.agent` service, and should be run under the logged-in user. For these commands, you can also use the Datadog Agent systray app
 * all the other commands can still be run with the `datadog-agent` command (located in the `PATH` (`/usr/local/bin/`) by default)
 * the `info` command has been renamed `status`
+* The configuration GUI is now a web-based configuration application, point a
+web browser to `http://localhost:5002` to access it.
```

Comment:
we could also mention running `datadog-agent launch-gui` directly from a shell and using `Open Web UI` on the systray app. Easier to remember than the port IMO :)